erpnext/accounts/doctype/pos_closing_entry/pos_closing_entry.js

```diff
@@ -128,7 +128,7 @@ function refresh_payments(d, frm, remove) {
 	d.payments.forEach(p => {
 		const payment = frm.doc.payment_reconciliation.find(pay => pay.mode_of_payment === p.mode_of_payment);
 		if (payment) {
-			if (!remove) payment.expected_amount += flt(p.amount);
+			if (!remove) payment.expected_amount += flt(d.grand_total);
```

Comment:
Hey @mcgun182. This won't work for invoices paid by multiple modes. For example, if an invoice of grand total 1500 is paid via 1000 Cash and 500 Credit Card, then expected amount of Cash & Credit Card both will be set as 1500.

Reply:
I've made an adjustment, will that be a better solution?